app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -0,0 +1,73 @@
+<!--
+  ~ Copyright (c) 2021 2bllw8
+  ~ SPDX-License-Identifier: GPL-3.0-only
+  -->
```

Comment:
As the translation is provided primarily by you, I think we can do away with this copyright notice. Feel free to remove it entirely, or add your name and current year.

Reply:
Resolved, and thanks!